cypress/integration/Regression/CreateModels/AllEntityTypes2.spec.js

```diff
@@ -0,0 +1,38 @@
+/**
+* Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+*/
+
+import * as models from '../../../support/Models'
+import * as entities from '../../../support/Entities'
+import * as helpers from '../../../support/Helpers'
+
+describe('All Entity Types 2 - CreateModels', () => {
+  afterEach(helpers.SkipRemainingTestsOfSuiteIfFailed)
+
+  context('Setup', () => {
+    it('Should create a model to test against', () => {
+      models.CreateNewModel('z-allEntityTypes2')
+    })
+  })
+
+  // These tests differ from the tests in AllEntityTypes1 by reversing the order of creating the Entities
+  // with resolver type and pretrained, along with adding an extra resolver type entity. The purpose is to
+  // verify that we don't get the pop-up (after the first of that type) that says...
+  //    Note: You must wait for training to complete before the new pre-trained Entity will be detected
+  // Also we want to verify all combinations of multivalue and negatable resolver types can be created.
+  context('Create Entities', () => {
+    entities.pretrainedEntityTypes.forEach(entityType => { 
```

Comment:
I didn't know if you planned to add them in separate PR which is fine, but we recently added `geographyV2`, `personName`, and `keyPhrase` as new types.

You would need to update the `pretrainedEntityTypes` list if you want to test those although it doesn't matter much as testing path of one is identical to another except the name.